schema/deploy/shipping/views.sql

```diff
@@ -223,4 +225,15 @@ grant select
    on shipping.incidence_model_observation_v2
    to "incidence-modeler";
 
+
+create view shipping.observation_with_presence_absence_result as
+
+    select target,
+           present,
+           present::int as presence,
+           observation.*
+      from shipping.incidence_model_observation_v2 as observation
+      join shipping.presence_absence_result_v1 using (sample)
+      order by site, encounter, sample, target;
```

Comment:
Do we want to user an inner `join` here or something else? I know this is how @tsibley originally had it, but I just wanted to know the reasoning behind the choice.

Reply:
I was under the impression that this view will display only encounters that have presence/absence results, but I may be wrong.

Reply:
Yep, I'm just curious to know why we would only want to see only encounters that have presence/absence results rather than showing all encounters and having the user optionally filter to exclude `null` presence/absence.

Reply:
Good question to consider! :-) I believe the reason why I didn't make it a left join was that the only purpose of the view was to join up encounters with presence/absence for convenience. It seemed that nearly all use cases would want to add the filter, so I made it part of the view.

I guess the question is in what cases would we a) want to use this view and b) want to include encounters without p/a results?  Perhaps one way to answer this is to look at how the draft view is currently used in Metabase.

Reply:
It looks like all questions using the draft view are looking at encounters that have presence/absence results: https://backoffice.seattleflu.org/metabase/reference/databases/3/tables/66/questions

Reply:
Ok! Let's leave this as an inner join then.